src/api/spec/models/obs_factory/staging_project_spec.rb

```diff
@@ -39,4 +53,623 @@
       it { expect(subject.adi_staging?).to be false }
     end
   end
+
+  describe 'self.for' do
+    context 'openSUSE:Factory' do
+      subject { ObsFactory::StagingProject.for(factory_distribution) }
+
+      it { expect(subject.count).to eq(3) }
+    end
+
+    context 'openSUSE:Factory:PowerPC' do
+      let(:factoryppc_distribution) { ObsFactory::Distribution.find(factoryppc.name) }
+      subject { ObsFactory::StagingProject.for(factoryppc_distribution) }
+
+      it { expect(subject.count).to eq(3) }
+    end
+
+    context 'openSUSE:13.2' do
+      let(:opensuse_132_distribution) { ObsFactory::Distribution.find(opensuse_132.name) }
+      subject { ObsFactory::StagingProject.for(opensuse_132_distribution) }
+
+      it { expect(subject.count).to eq(2) }
+    end
```

Comment:
I find it quite hard to understand what #for does by reading through this test. Does the behaviour of this method change when using  13.2 instead of factory ppc?
If not I would use the same distro and make it more visible what causes the count / result list to change. If I understand correctly it's the subprojects